docs/2023/rest/updates/samuel/2023-06-15.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Week 3
+author: Samuel Dushimimana
+author_url: https://github.com/dushimsam
+author_image_url: https://avatars.githubusercontent.com/u/66276301?v=4
+tags: [gsoc23, rest]
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2023 Samuel Dushimimana <dushsam100@gmail.com>
+-->
+
+
+## Activities
+
+* Added the REST API to remove a particular main license from the upload: [#2463](https://github.com/fossology/fossology/pull/2463)
+* Added the REST API to retrieve all the main licenses assigned on a particular upload: [#2465](https://github.com/fossology/fossology/pull/2465)
+* Added the REST API to add new license decision for an item: [#2466](https://github.com/fossology/fossology/pull/2466)
+* Added the REST API to delete a specific license decision assigned on an item:  [#2469](https://github.com/fossology/fossology/pull/2469)
+* Added the REST API to list the license decisions for a specific item:  [#2470](https://github.com/fossology/fossology/pull/2470)
+* Added the REST API to update the clearing info (**License text**, **acknowledgement** & **Comment**) for a specific license decision on the item:  [#2471](https://github.com/fossology/fossology/pull/2471)
+* Added the REST API to retrieve the clearing history for a specific item: [#2477](https://github.com/fossology/fossology/pull/2477)
+
+### SOME SCREENSHOTS
+
+#### 1. REMOVE MAIN LICENSE API `/uploads/{id}/licenses/{licenseId}/main`
+
+![delete-main-license](/img/reactUI/api/License/delete_main_license.png)
+
+#### 2. GET MAIN LICENSES API `/uploads/{id}/licenses/main`
+
+![get-main-licenses](/img/reactUI/api/License/get_main_licenses.png)
+
+#### 3. ADD NEW LICENSE DECISION API `/uploads/{id}/items/{itemId}/licenses`
+
+![add-license-decision](/img/reactUI/api/License/add_new_license_decision.png)
+
+#### 4. DELETE LICENSE DECISION API `/uploads/{id}/items/{itemId}/licenses/{licenseId}`
+
+![delete-license-decision](/img/reactUI/api/License/delete_license_decision.png) 
+
+
+#### 5. GET ALL LICENSE DECISIONS FOR AN ITEM API `/uploads/{id}/items/{itemId}/licenses`
+
+![get-license-decisions](/img/reactUI/api/License/get_all_license_decisions.png) 
+
+#### 6. UPDATE THE CLEARING INFO API `/uploads/{id}/items/{itemId}/licenses`
+
+![update-license-decision](/img/reactUI/api/License/set_clearing_info.png)
+
+#### 7. GET CLEARING HISTORY API `/uploads/{id}/items/{itemId}/clearing_history`
+
+![get-clearing-history](/img/reactUI/api/License/get_clearing_history.png)
+
+## Next Week's Expectations
+
+- Add the PR for the REST API to schedule the bulk scan.
+- Add the PR for the REST API to handle the three-pointers functionality.
+- Add the REST API to retrieve the positions for the highlighting the content of the file.
+- Add the REST API for the Tree view: Reuse Summary, Scanner Details (GET & for filter)	
+
+
+
+
+
+
```

Comment:
Properly format your markdown files. You can use the inbuilt VS Code formatter. Or install the Markdown Lint extension.

Reply:
Great